wiki/Beatmapping/Health_Drain/en.md

```diff
@@ -0,0 +1,7 @@
+# Health drain
+
+Health drain refers to the natural drain of the health bar that occurs when playing a [beatmap](/wiki/Beatmaps) while disregarding the influence of [hit objects](/wiki/Hit_Objects). This drain is dependent on the HP difficulty setting, which is set by the mapper of a beatmap. This value ranges from 0 to 10, where 10 is the highest and most punishing setting and 0 is the most forgiving. The HP difficulty setting can be changed alongside other difficulty settings in the [Beatmap Editor](/wiki/Beatmap_Editor). Doing this will change a Ranked beatmap's status to unranked and will erase all local scores on it.
+
+osu!standard and osu!catch have health drain at a constant rate throughout the course of a beatmap, with the exception of breaks. As for osu!taiko and osu!mania, health drain is disabled, so only the hit objects will affect the health bar.
```

Comment:
> As for osu!taiko and osu!mania, health drain is disabled, so only the hit objects will affect the health bar.
Kinda makes it sound like HP doesn't matter, might be better worded as "In osu!taiko and osu!mania, health drain occurs when an object is not successfully hit."

Reply:
yea, still need to change some things regarding misses and geki/katu health gain. Still unsure if i should split it up more and put the full HP description under the health bar article. Or do the opposite and link to hpdrain from UI health bar article once that exists

Reply:
probably going to end up doing the latter of those two.

Reply:
imo latter option is better, this article seems more about the mechanics and the UI article would be about how it's displayed& skinned